webpack.config.ts

```diff
@@ -38,7 +43,7 @@ const config: webpack.Configuration = {
     }
   },
   node: {
-    fs: false,
+    fs: "empty",
```

Comment:
This lets us just ignore fs in the browser environment--as long as we don't actually run a line of code which uses fs, we're all good.